jetty-servlets/src/main/java/org/eclipse/jetty/servlets/DoSFilter.java

```diff
@@ -345,15 +347,15 @@ protected void doFilter(HttpServletRequest request, HttpServletResponse response
                 case 0:
                 {
                     // Fall through to throttle the request.
```

Comment:
The comment says we should fall through to throttle the request, but we are not!!   Comments should be "break to throttle..."

Reply:
We do fall through .. the entire switch statement into the throttling code.

The other switch cases (even default) return.

Reply:
fall through normally means from one case to the next.   We break the switch to get to the throttling code.
But this switch will change entirely if my listener suggestion works :)